andela/ah-backend-thor
# #[161382394] Users should be able to create their profiles

#### What does this PR do?
Compares ft-create-user-profile-161382394 btanch with develop
#### Description of Task to be completed?
Users profiles are created on registration and users can update their profiles
#### How should this be manually tested?
In postman,  create an account, verify the account using the email link, log in and get a token use the token to access the user endpoint and pass the data below
{
  "user":{
    "bio": "body",
    "image": "https://kegz.imgix.net/hp/snowshoe.jpg?auto=compress&w=900&h=600&fit=crop",
    "username":"jojo3"
  }
}
#### Any background context you want to provide?
Users should get a profile on successfull registration.
#### What are the relevant pivotal tracker stories?
161382394
